src/view/RegisterUser.java

```diff
@@ -10,6 +10,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.sql.SQLException;
```

Comment:
Explain this line, why did you import it?

Reply:
This line is here because we have called the Login class. And in the login class, that's where we are creating our initial and only connection with the database, as I said in the title of this PR, I wrote cleaner, faster, and shorter code by removing the need to initiate or create a database connection each time we make a request to the DB. We shall create the connection to the DB once, in the Login class, and then make requests to the DB using that same connection. That's where we need to throw the SQLException in case the connection fails. Now each time we make a request, we will need to throw the SQLException so that we can catch connection errors. Each class that instantiates the Login class is going to also throw the SQLException as a result of what I just stated above.